src/ocplib_i18n.ml

```diff
@@ -10,7 +10,7 @@
  * The above copyright notice and this permission notice shall be included in all
  * copies or substantial portions of the Software. *)
 
-let known_lang_ids = [%lang_ids_array]
+let known_lang_ids = (* [%lang_ids_array] *) [| "en"; "fr" |]
```

Comment:
hmm, indeed, my solution was intended to be embedded, but implied that you needed the translation files before compiling the lib, which can not work if it's a separate package.
One solution could be to be rid of the lib entirely, and replace `set_lang` by an inline version generated by the ppx.

But indeed, I think the best approach would be to combine the ppx (for translated strings discovery and generation/update of `.po` translation files) with a less trivial lib that could do dynamic discovery of translations rather than just a static array access.

If made dynamic, we should probably make the «parsing» of the `.po` files more robust though.